pkg/routes/v1.go

```diff
@@ -14,14 +14,17 @@ func loadV1Routes(r *gin.Engine, h *handler.Handler, cfg *config.Config) {
 	// employees
 	v1.GET("/employees", mw.WithAuth, mw.WithPerm(cfg, "employees.read"), h.Employee.List)
 	v1.GET("/employees/:id", h.Employee.One)
-	v1.POST("/employee/:id/employee-status", mw.WithAuth, h.Employee.UpdateEmployeeStatus)
-	v1.GET("/employee/profile", mw.WithAuth, h.Employee.GetProfile)
+	v1.POST("/employees/:id/employee-status", mw.WithAuth, h.Employee.UpdateEmployeeStatus)
+	v1.GET("/employees/profile", mw.WithAuth, h.Employee.GetProfile)
+
+	v1.POST("/employees/:id/edit", mw.WithAuth, h.Employee.Update)
```

Comment:
don't add action to path (ref: https://stackoverflow.blog/2020/03/02/best-practices-for-rest-api-design/#h-use-nouns-instead-of-verbs-in-endpoint-paths)

Reply:
```suggestion
	v1.PUT("/employees/:id", mw.WithAuth, h.Employee.Update)
```

Reply:
thank you, I chaged it